docs/volcano_integration.md

```diff
@@ -0,0 +1,107 @@
+# Integration with Volcano for Batch Scheduling
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workloads.
+With the integration with Volcano, Flink job and task managers can be scheduled for better scheduling efficiency.
+
+## Prerequisites
+
+## Install Volcano
+
+- Install from provided demo
+
+Run the following 
+```bash
+kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
+```
+
+- Install with advanced settings
+
+Please refer to [Volcano Official Guide](https://volcano.sh/docs/getting-started/)
+ 
+## Install Flink Operator with Volcano enabled
+
+## Installing the Chart
+
+1. Prepare a Flink operator image. Follow the instructions [here](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/developer_guide.md#build-and-push-docker-image) to build and push an image from the source code.
+
+2. Run the bash script `update_template.sh` to update the manifest files in templates from the Flink operator source repo (This step is only required if you want to install from the local chart repo).
+
+3. Register CRD - Don't manually register CRD unless helm install below fails (You can skip this step if your helm version is v3). 
+    
+    ```bash
+   kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/flink-on-k8s-operator/master/config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml
+   ```
+
+4. Finally operator chart can be installed by running:
+
+	```bash
+	helm repo add flink-operator-repo https://googlecloudplatform.github.io/flink-on-k8s-operator/
+	helm install --name [RELEASE_NAME] flink-operator-repo/flink-operator --set operatorImage.name=[IMAGE_NAME]
+	```
+    or to install it using local repo with command:
+
+    ```bash
+    helm install --name [RELEASE_NAME] . --set operatorImage.name=[IMAGE_NAME] --set enableBatchScheduler=true
```

Comment:
Add commands on how to verify Valcano is up and running, after the installation.